soc/intel/intel_adsp/Kconfig.defconfig

```diff
@@ -48,4 +48,7 @@ choice CACHE_TYPE
 	default ARCH_CACHE
 endchoice
 
+config DCACHE_LINE_SIZE
+	default 64
+
```

Comment:
It would help to explain this is currently defined in the board files:

boards/intel/adsp/intel_adsp_ace15_mtpm_defconfig:CONFIG_DCACHE_LINE_SIZE=64

... and friends. So this is more about moving the cache size definition from board to soc level (which agreed is more appropriate).